src/test/java/org/takes/facets/hamcrest/HmRsStatusTest.java

```diff
@@ -67,8 +67,8 @@ public void responseCodeIsNotEqualToExpected() throws IOException {
         MatcherAssert.assertThat(
             new TkHTML("<html><body/></html>").act(new RqFake()),
             Matchers.not(
-                Matchers.is(
```

Comment:
@erimerturk this method should be renamed to `testsStatusNotFound` and javadoc should read `HmRsStatus can test status HTTP_NOT_FOUND`